app/hub/Views/SideNavigationView/SideNavigationViewContainer.jsx

```diff
@@ -51,10 +52,9 @@ class SideNavigationViewContainer extends Component {
 			{ href: '/setup', icon: 'setup', text: t('hub_side_navigation_setup') },
 			{ href: '/tutorial', icon: 'tutorial', text: t('hub_side_navigation_tutorial') },
 			{ href: '/plus', icon: 'plus', text: t('hub_side_navigation_supporter') },
-			{ href: '/rewards', icon: 'rewards', text: t('hub_side_navigation_rewards') },
-			{ href: '/products', icon: 'products', text: t('hub_side_navigation_products') }
+			...(IS_CLIQZ ? [] : [{ href: '/rewards', icon: 'rewards', text: t('hub_side_navigation_rewards') }]),
+			...[{ href: '/products', icon: 'products', text: t('hub_side_navigation_products') }]
```

Comment:
you can simplify the last entry to
`{ href: '/products', icon: 'products', text: t('hub_side_navigation_products')}`
(i.e. you don't need the array destructuring syntax for this element)

Reply:
(@fcjr fyi - didn't notice this when you were asking me about it the other day)

Reply:
Great catch. I changed it.